process/charter.html

```diff
@@ -214,52 +214,52 @@ <h3 id="recharter-request">Request for rechartering</h3>
 </dd>
 </dl>
 
-<h2 id="implementation-details">Implementation Details</h2>
+<h2 id="implementation-details">Implementation details</h2>
 
 <p>The following sections are mostly intended as instructions to the Team and are included here for transparency.</p>
 
 <h3 id="baseline-support">Sufficient Member support for a charter</h3>
 
 <p>
 Generally, the Team will expect to receive reviews for Charter
-proposals from at least 5% of the Membership. If this 5% threshold is
+proposals from at least 5% of the <a href="/membership/list/">Membership</a>. If the 5% threshold is
 not met, the Charter may still be approved, but additional scrutiny is
 warranted, and resource allocation may be limited. Additionally, the
 Team will continue to consider the number of declarations of intent to
-participate or implement the output of the Working Group.</p>
+participate or implement the output of the work Group.</p>
 
 <p>Given the diversity of work underway in the Consortium, including
-groups that focus on horizontal reviews (e.g. accessibility, security,
+groups that focus on horizontal reviews (e.g., accessibility, security,
 and internationalization), as well as technologies that are initially
 focused on by some segment of the Web's stakeholders, there are times
-where exceptions to this practice may be warranted. In those cases,
-the Team will document reasons why the exception is warranted.</p>
+where exceptions to this practice may be considered In those cases,
+the Team will document reasons why the exception is made</p>
 
 <h3 id="speed-tips">Tips to speed up the process</h3>
 
 <p>Parallelize where possible:</p>
 
     <ul>
-      <li>Work with W3C Communications Team to send <a href="adv-notice.html">advance notice to AC</a> (via the <a href="#pipeline">pipeline</a>) as soon as you have something to point to.</li>
+      <li>Work with the W3C Communications Team to send <a href="adv-notice.html">advance notice to the AC</a> as soon as you have something to point to, and update the <a href="#pipeline">pipeline</a>.</li>
       <li>Start the horizontal reviews of the charter as soon as the scope, deliverables, and dependencies
          parts of the charter are stable enough. Ping the horizontal reviewers on a weekly basis.</li>
-      <li>Start working on resolving any formal objection on a charter as soon as those received. Don't wait
-        for the end of the AC review period.
+      <li>Start working on resolving Formal Objections to a charter as soon as those are received. Don't wait
+        till the end of the AC review period.
       </li>
-      <li>Prepare a draft of your proposed W3C approval announcement before getting the approval from TiLT</li>
+      <li>Prepare a draft of the W3C Desicion announcement before getting the approval from TiLT</li>
```

Comment:
```suggestion
      <li>Prepare a draft of the W3C Decision announcement before getting the approval from TiLT</li>
```

Reply:
Also, can "W3C Decision" be more precise e.g., whether that's W3C *Team Decision* (as per Process) or W3C-wide (including Team, Members, AB, ..)? The context is not entirely clear to me, i.e., is it related to the responses to the FO or other change requests or a final decision.

Reply:
It’s the “W3C Decision” as defined in the Process (so one that results from Members review through a questionnaire).

Reply:
I completely overlooked that, thanks!